.gitignore

```diff
@@ -1,6 +1,7 @@
 **/node_modules/**
 npm-debug.log
 coverage/
+enforcer.sh
```

Comment:
This shouldn't be here.

Reply:
This script is downloaded with each run (to keep it up to date) - so it shouldn't be checked in to the repo.

Reply:
But it's downloaded to CI, not locally.

Reply:
I don't feel strongly about this, so I'm fine reverting.